### ISSUE TITLE
*.test.js & *.js in different casings

### ISSUE THREAD

**Issue by imlinus** [loeffel-io/ls-lint#3]
First off, what a great library. I'm totally loving this 👍 👍 

I don't know if I really can call this a issue, _it's rather me and my old codebase_ that's not up to par (I should probably have them in a separate `/tests` folder)

I've got a bunch of .vue-files in `PascalCase.vue` and some tests `PascalCase.test.js` right next to them (in the same folders)

I would like the rest of my "pure" .js-files to be in `camelCase.js`

Is this possible? 👼 

Cheers,
Linus

**Comment by loeffel-io** [comment]
Hey @imlinus, 

thank you so much, love to hear that! ❤️ 

Currently there is no support for different cases in the same directory. 
There could be an _or_ logic operator to provide multiple rules for the same directory, but this would just crush up your project structure. 

For example, i could not find any [google ](https://github.com/google) open source project with different cases in the same directory - They always have a clean structure and with [ls-lint](https://github.com/loeffel-io/ls-lint) i try to bring this clean project structures to all of your projects. 

For your case, i would recommend to move your test files in a new subdirectory without the `.test` suffix like:

```yaml
# .ls-lint.yml

ls:
  .js: camelCase
  .vue: PascalCase

  tests/components:
    .js: PascalCase

ignore:
  - node_modules
```

**Comment by imlinus** [comment]
You're absolutely right, `*.test.js` is an old recommendation from Jest, the new preferred way is to separate them by folder, so that's what I'll do :-D

Cheers,
Linus

**Comment by loeffel-io** [comment]
fyi @imlinus: #5 will add support for sub extensions to add more flexibility

**Comment by imlinus** [comment]
Wonderful ❤️

**Comment by loeffel-io** [comment]
Will be available in a few minutes in npm package v1.2.1 🚀

**Comment by imlinus** [comment]
I'll try it out right away 💯
